webpack.config.js

```diff
@@ -48,33 +49,68 @@ module.exports = {
             'amd': 'fabric',
             'root': 'fabric'
         }
-    },
+    }],
     module: {
-        preLoaders: [
+        rules: [
             {
                 test: /\.js$/,
-                exclude: /(node_modules|bower_components)/,
-                loader: 'eslint-loader'
-            }
-        ],
-        loaders: [
-            {
+                exclude: /node_modules|bower_components/,
```

Comment:
bower?

Reply:
I knew wrong usage, bower did not need to excloud. OK